components/wpt-results.html

```diff
@@ -248,64 +218,70 @@
       }
 
       _queryChanged () {
-        this._refreshDisplayedSpecDirs()
+        this._refreshDisplayedNodes()
       }
 
-      _refreshDisplayedSpecDirs () {
+      _refreshDisplayedNodes () {
         /* Recomputes the list of displayed spec directories and their child test results. */
```

Comment:
In this method comment: "spec directories" -> "nodes"

In general, is a node always a directory or can it also be a full test path?